test/functional/specs/desktop/C2584.js

```diff
@@ -16,42 +19,36 @@ test.meta({
   TEST_RUN: "Regression"
 });
 
-test("Test C2584: debug command with enable: true. getLibraryInfo. refresh. toggle and repeat.", async t => {
-  const logger = createConsoleLogger(t, "log");
-  await t
-    .click(Selector("#nologconfig-button"))
-    .click(Selector("#debugtrue-button"))
-    .click(Selector("#getlibraryinfo-button"));
+const debugCommand = ClientFunction(enabled => {
+  return new Promise(resolve => {
+    window.alloy("debug", { enabled }).then(() => resolve());
```

Comment:
Why are you wrapping this in an extra promise?

Reply:
To make `ClientFunction` async to prevent any chances of race conditions. Might not be 100% needed, I just got in the habit of wrapping ALL commands :)

Reply:
Alloy commands already return promises.  I don't think you need the wrapping.